quesma/queryparser/pancake_model.go

```diff
@@ -28,15 +29,17 @@ type pancakeAggregationLevel struct {
 }
 
 type pancakeFillingMetricAggregation struct {
-	name            string
+	name            string          // as originally appeared in Query DSL
+	aliasName       string          // full name with path, e.g. metric__byCountry__byCity__population or aggr__byCountry
```

Comment:
I'm wondering whether `aliasName` is the best name for that, maybe `internalName`?

Reply:
Or `mangledName`

Reply:
Changed to `internalName`